instrumentation/httpclient5/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright 2013-2020 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.zipkin.brave</groupId>
+    <artifactId>brave-instrumentation-parent</artifactId>
+    <version>5.13.3-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>brave-instrumentation-httpclient5</artifactId>
+  <name>Brave Instrumentation: Apache HttpClient v5.0+</name>
+
+  <properties>
+    <!-- Matches Export-Package in bnd.bnd -->
+    <module.name>brave.httpclient5</module.name>
+
+    <main.basedir>${project.basedir}/../..</main.basedir>
+    <main.java.version>1.7</main.java.version>
+    <main.signature.artifact>java17</main.signature.artifact>
+    <httpclient5.version>5.0.3</httpclient5.version>
```

Comment:
Add XML comment here saying 5.0.4 is needed and ref the two issues. That comment should stay after we update so that people know (in case someone changes version and has a problem and no one remember)

Reply:
@kojilin do you think the comments suggested here are worth adding?

Reply:
I added the comment we need to use >= 5.2